cart/cart.css

```diff
@@ -0,0 +1,172 @@
+.card {
```

Comment:
![PR review](https://user-images.githubusercontent.com/89376505/154833553-bc5178b0-f4a7-48c6-a44f-b304fda3e0f3.JPG)

Just a small suggestion
You can increase the height of the card so that the button looks like a part of the card!!